salesforce/salesforce-inbound-endpoint/salesforce-inbound-endpoint-1.0.0/src/main/java/org/wso2/carbon/inbound/salesforce/poll/SalesforceConstant.java

```diff
@@ -0,0 +1,62 @@
+/*
+ * Copyright (c) 2015, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ *   WSO2 Inc. licenses this file to you under the Apache License,
+ *   Version 2.0 (the "License"); you may not use this file except
+ *   in compliance with the License.
+ *   You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing,
+ *  software distributed under the License is distributed on an
+ *  "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ *  KIND, either express or implied.  See the License for the
+ *  specific language governing permissions and limitations
+ *  under the License.
+ */
+package org.wso2.carbon.inbound.salesforce.poll;
+
+public class SalesforceConstant {
+
```

Comment:
If this is a constant class then better to have a private constructor to avoid creation of sub classes.